nemo/collections/asr/parts/submodules/ctc_beam_decoding.py

```diff
@@ -191,8 +203,10 @@ def __init__(
             self.search_algorithm = self.default_beam_search
         elif search_type == "pyctcdecode":
             self.search_algorithm = self._pyctcdecode_beam_search
-
+            
             raise NotImplementedError(f"The search type of `pyctcdecode` is currently not supported.\n" f"")
+        elif search_type == "flashlight":
+            self.search_algorithm = self._flashlight_beam_search
```

Comment:
No need for _ in function name, it can be public method.

Reply:
Fixed